datasets/senti_ws/senti_ws.py

```diff
@@ -0,0 +1,133 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""SentiWS: German-language resource for sentiment analysis, pos-tagging"""
+
+from __future__ import absolute_import, division, print_function
+
+import os
+
+import datasets
+
+
+_CITATION = """\
+@INPROCEEDINGS{remquahey2010,
+title = {SentiWS -- a Publicly Available German-language Resource for Sentiment Analysis},
+booktitle = {Proceedings of the 7th International Language Resources and Evaluation (LREC'10)},
+author = {Remus, R. and Quasthoff, U. and Heyer, G.},
+year = {2010}
+}
+"""
+
+_DESCRIPTION = """\
+SentimentWortschatz, or SentiWS for short, is a publicly available German-language resource for sentiment analysis, and pos-tagging. The POS tags are ["NN", "VVINF", "ADJX", "ADV"] -> ["noun", "verb", "adjective", "adverb"], and positive and negative polarity bearing words are weighted within the interval of [-1, 1].
+"""
+
+_HOMEPAGE = ""
+
+_LICENSE = "Creative Commons Attribution-Noncommercial-Share Alike 3.0 Unported License"
+
+_URLs = ["https://pcai056.informatik.uni-leipzig.de/downloads/etc/SentiWS/SentiWS_v2.0.zip"]
+
+
+class SentiWS(datasets.GeneratorBasedBuilder):
+    """SentiWS: German-language resource for sentiment analysis, pos-tagging"""
+
+    VERSION = datasets.Version("1.1.0")
+
+    BUILDER_CONFIGS = [
+        datasets.BuilderConfig(name="pos-tagging", version=VERSION, description="This covers pos-tagging task"),
+        datasets.BuilderConfig(
+            name="sentiment-scoring",
+            version=VERSION,
+            description="This covers the sentiment-scoring in [-1, 1] corresponding to (negative, positive) sentiment",
+        ),
+    ]
+
+    DEFAULT_CONFIG_NAME = "pos-tagging"
+
+    def _info(self):
+
+        if (
+            self.config.name == "pos-tagging"
+        ):  # the pos-tags are ["NN", "VVINF", "ADJX", "ADV"] -> ["noun", "verb", "adjective", "adverb"]
+            features = datasets.Features(
+                {
+                    "word": datasets.Value("string"),
+                    "pos-tag": datasets.Value("string"),
```

Comment:
```suggestion
                    "pos_tag": datasets.ClassLabel(names=["NN", "VVINF", "ADJX", "ADV"]),
```
can you use ClassLabel here please and rename the column for consistency with other datasets ?